packages/fiber/tests/web/canvas.test.tsx

```diff
@@ -25,6 +25,20 @@ describe('web Canvas', () => {
     expect(renderer.container).toMatchSnapshot()
   })
 
+  it('should set ref', async () => {
+    const ref = React.createRef<HTMLCanvasElement>()
+
+    await act(async () => {
+      render(
+        <Canvas ref={ref}>
+          <group />
+        </Canvas>,
+      )
+    })
+
+    expect(ref).toBeDefined()
```

Comment:
shouldn't this be `expect(ref.current).toBeDefined()`?

Reply:
oh duh... thanks!